symphony/lib/toolkit/class.widget.php

```diff
@@ -79,7 +79,7 @@ public static function Input($name, $value = null, $type = 'text', array $attrib
     {
         General::ensureType(array(
             'name' => array('var' => $name, 'type' => 'string'),
-            'value' => array('var' => $value, 'type' => 'string', 'optional' => true),
+            'value' => array('var' => $value, 'type' => ['string', 'int', 'double', 'float'], 'optional' => true),
```

Comment:
`double` is an alias of `float`, was there a benefit to including both instead of just `float`?

Reply:
Yup, good point! I didn't review that array after I wrote it the first time.